Knossos.NET/Models/CustomLauncher.cs

```diff
@@ -164,6 +179,11 @@ public static class CustomLauncher
         /// </summary>
         public static LinkButton[]? HomeLinkButtons { get; private set; }
 
+        /// <summary>
+        /// Costumisable data for home screen buttons
```

Comment:
super tiny thing, but should this be `Customisable`?

Reply:
y